proctord/config/config.go

```diff
@@ -60,6 +60,11 @@ func KubeJobActiveDeadlineSeconds() *int64 {
 	return &tmp
 }
 
+func KubeJobRetries() *int32 {
+	tmp := int32(viper.GetInt("KUBE_JOB_RETRIES"))
```

Comment:
can we rename `tmp` to more meaningful?

Reply:
It's actually a temporary variable. The only reason it exists is you can't instantiate a pointer literal in golang. You need to assign it to a variable and reference the variable to pass a pointer.